test/test_other.py

```diff
@@ -9561,6 +9561,18 @@ def test_wasm_sourcemap_dead(self):
     # has only two entries
     self.assertRegexpMatches(output, r'"mappings":\s*"[A-Za-z0-9+/]+,[A-Za-z0-9+/]+"')
 
+  def test_wasm_sourcemap_resolve_dummy_comp_dir(self):
+    wasm_map_cmd = [PYTHON, path_from_root('tools/wasm-sourcemap.py'),
+                    '--dwarfdump-output',
+                    test_file('other/wasm_sourcemap_resolve_dummy_comp_dir/foo.wasm.dump'),
+                    '-o', 'a.out.wasm.map',
+                    test_file('other/wasm_sourcemap_resolve_dummy_comp_dir/foo.wasm'),
```

Comment:
Can you use use just compile `test/hello_world.c` here rather then checking in the `foo.wasm` binary?

Also it looks like you accidentally added the `.dump` file to git too.